newsboat.cpp

```diff
@@ -131,15 +137,17 @@ void print_usage(const std::string& argv0, const ConfigPaths& configpaths)
 		}
 		longcolumn += "--" + a.longname;
 		longcolumn += a.params.size() > 0 ? "=" + a.params : "";
-		ss << "\t" << longcolumn;
-		for (unsigned int j = 0; j < utils::gentabs(longcolumn); j++) {
-			ss << "\t";
-		}
-		ss << a.desc << std::endl;
-	}
-	std::cout << ss.str();
 
-	std::cout << '\n';
+		maxLength = std::max(maxLength, longcolumn.length());
+		helpLines.push_back({longcolumn, a.desc});
+	}
+	for (const auto& helpLine : helpLines) {
+		std::cout << std::string(8, ' ') << helpLine.first;
+		const auto padding = maxLength - helpLine.first.length();
+		std::cout << std::string(1 + padding, ' ') << helpLine.second;
```

Comment:
This must be the ultimate nitpick, but I'd prefer to have two spaces between the option and its description -- this better separates the two.

```suggestion
		std::cout << std::string(2 + padding, ' ') << helpLine.second;
```